### ISSUE TITLE
VSTS extension not working with Excel

### ISSUE THREAD

**Issue by KelBowen** [snuziale/vsts-open-in-excel#11]
I adore this extension but now I am very sad as it has stopped working. See the following screen shots. The last picture gives you the version of Excel I'm running. 
![greyedout](https://user-images.githubusercontent.com/5777019/35300094-534f0a32-003c-11e8-81a1-46149fe0e54e.JPG)
![error1](https://user-images.githubusercontent.com/5777019/35300095-537cd0d4-003c-11e8-876f-28d6fe8cd5bc.JPG)
![error2](https://user-images.githubusercontent.com/5777019/35300096-53ca8ce8-003c-11e8-81cc-688c61d6bf12.JPG)
![excelversion](https://user-images.githubusercontent.com/5777019/35300097-53decb72-003c-11e8-80af-9d6fcf6e491e.JPG)

**Comment by nschonni** [comment]
Would that be an issue with the TFS Office Integration rather than this plugin https://www.visualstudio.com/downloads/#team-foundation-server-office-integration-2017 (or the version shipped with your VS)? AFAIK, this plug-in only generates the query+link that the TFS client side COM plug-in then intercepts.

PS: I see you're running a "Dogfood" version of Excel, so maybe something with the TFS integration is incompatible with a new API

**Comment by KelBowen** [comment]
Yes, I'm assuming that my issue is probably around the dogfood version of Excel. I'm just not sure who to inform. I was troubleshooting my environment - enabling/disabling the add-in, ran a repair on Office. A co-worker is experiencing the same thing so, I'm assuming there must be some incompatibility between TFS and the new API. Trying to figure out how to report it to someone. Thanks for the quick reply. Any suggestions for what I should do next?

**Comment by nschonni** [comment]
It may seem silly since it's your own company, but my 2 cents would be to file something here https://developercommunity.visualstudio.com/spaces/22/index.html
That way it might get triaged to the right team in MS that owns the TFS Office integration
